### PR TITLE
Handle data sample empty cases

### PR DESCRIPTION
**Pull request by albertoac**

### CODE REVIEW

File: do-catalog/src/styles/components/_buttons.scss [hunk 1]
```diff
@@ -1,112 +1,112 @@
...
-    }
-  }
-
-  &.buttonLink--center {
-    width: 100%;
-    text-align: center;
-  }
-}
+// .button {
+//   display: flex;
```
Comment (neokore) [inline do-catalog/src/styles/components/_buttons.scss:2]: If it will not be used, better comment its import or delete it.

Comment (albertoac) [reply do-catalog/src/styles/components/_buttons.scss]: Forgot to remove the file 😅

File: do-catalog/src/components/Button.vue [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  padding: 10px 16px;
+  border-radius: 4px;
+  background: $button__bg-color;
+  color: $button__color;
+  font: 600 12px/16px $title__font-family;
+  letter-spacing: 0;
+  cursor: pointer;
+
+  &:hover {
+    filter: brightness(98%);
```
Comment (neokore) [inline do-catalog/src/components/Button.vue:34]: Better use the final color to keep compatibility with IE11 and company

File: do-catalog/src/utils/form-url.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+export function formUrl(category, country, datastream) {
+  var formUrl = "https://carto.com/platform/location-data-streams/form/?";
```
Comment (neokore) [inline do-catalog/src/utils/form-url.js:2]: `let` instead of `var`

File: do-catalog/src/utils/form-url.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+  formUrl = category ? formUrl + "category=" + category + "&" : formUrl;
```
Comment (neokore) [inline do-catalog/src/utils/form-url.js:3]:
```suggestion
  formUrl += category ? `category=${category}&` : '';
```

Comment (neokore) [reply do-catalog/src/utils/form-url.js:3]: Same for the rest, take a look to https://github.com/CartoDB/common-ui/blob/9bfed2a286fa88b468d069251cb67ef62f9420f7/do-catalog/src/store/actions/catalog.js#L32

File: do-catalog/src/views/DatasetData.vue [hunk 5]
```diff
@@ -76,9 +90,14 @@
 
 <script>
 import { mapState } from 'vuex';
+import Button from '@/components/Button.vue';
+import { formUrl } from '@/utils/form-url';
```
Comment (neokore) [inline do-catalog/src/views/DatasetData.vue:94]: I'm not sure about spliting this small function into another file.

Comment (albertoac) [reply do-catalog/src/views/DatasetData.vue:94]: As talked before, the formUrl function will be needed in other areas of the page, so is better to keep it separated

**Comment by Jesus89** [comment]
Could we close the `isPublicWebsite` method in the PR to complete it? How big /well-defined is this feature? @neokore

**Comment by Jesus89** [comment]
Let's cover the `isPublicWebsite` implementation in CH 69098

**Pull request merged**